docs/client-provisioning-methods.adoc

```diff
@@ -0,0 +1,40 @@
+// Copy of this topic: https://docs.ota.here.com/client-config/client-provisioning-methods.html adapted for aktualizr repo. Replaces duplicate "automatic-provisioning.adoc"
+
+In OTA Connect, the provisioning process ensures that the device has a certificate and a unique identifier. After a device has been provisioned, it can receive software updates. OTA Connect supports two types of provisioning:
+
+* *Provisioning with Shared Credentials*
+
+Devices use temporary provisioning credentials to request permanent credentials from the server. You download the temporary provisioning credentials from the OTA Connect web app and install them on a device. Once the device registers with the server for the first time, the server assigns permanent credentials to the device. The device then uses these permanent credentials for all future transactions.
+
+
+* *Provisioning with Device Credentials*
++
+Devices already have permanent credentials installed. The server doesn't issue any credentials to devices. Instead, you use a root CA certificate to sign the credentials that you install on the device. You then install the same root CA certificate on the OTA Connect server.
+Every time the device attempts to connect, the server verifies that the device credentials are signed by the same CA that you originally installed on the server.
+
+== Choosing a provisioning method
+
+The type of provisioning that you choose depends on your requirements. If you're just testing and want to get started quickly, provisioning with shared credentials is fine. It is easier to set up but is less secure. Because server self-signs the credentials that it issues to devices, the devices have no way of verifying the integrity of the server. This level of security is not suitable for a production environment. The device should always be able to verify that is communicating with a genuine OTA Connect server. Once you move to production, you should look at some form of provisioning that uses device certificates. For more security, use a Hardware Security Module (HSM) to store the device certificates.
+
+=== Configure a Device to Provision with Shared Credentials
+
+To configure a device to provision with shared credentials, follow these steps:
+
+. Download a https://docs.ota.here.com/quickstarts/generating-provisioning-credentials.html[provisioning key].
+. Edit the following `.toml` configuration file:
++
+link:https://github.com/advancedtelematic/aktualizr/blob/master/config/sota_autoprov.toml[config/sota_autoprov.toml]
+. Update the configuration file with the path to the zip file that you saved in the first step and start Aktualizr on the device.
```

Comment:
This isn't right. Editing that toml file in the aktualizr repo won't have any effect on bitbaking. meta-updater copies that file onto the device, but it takes it from whatever is in the aktualizr repo, not some locally-edited version. What you really need to do is tell meta-updater where to find your downloaded credentials by setting `SOTA_PACKED_CREDENTIALS` in your local.conf. If you really need to edit that file for bitbaking, there is a separate process, but I wouldn't go into that here. If you're just explaining how to run aktualizr locally, that's a separate matter, but that isn't made clear here, and you'd probably want to use a different config file to start with.

Reply:
Still a problem.